src/App.tsx

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './component/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
+class App extends React.Component {
```

Comment:
usually state is written in the same file

```suggestion
class App extends React.Component<{}, State> {
```